apps/docs/src/app/core/component-docs/wizard/examples/wizard-branching-example.component.html

```diff
@@ -112,7 +112,7 @@
 </div>
 
 <ng-template let-dialog let-dialogConfig="dialogConfig" #confirmationDialog>
-    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
+    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog" style="z-index: 1002;">
```

Comment:
Why did you add this? Will the user be required to add z-index in their project? If yes, we need to move this in fund-styles

Reply:
@InnaAtanasova Good question, I do not believe this will be necessary for the user to add.  It is because the z-index of the "wizard overlay" class is 1000, and the z-index of the fd-bar in the previous example is 1001 - so this is to ensure the dialog is over top of both the wizard overlay and the buttons from the fd-bar in the "mobile wizard" examples